### PR TITLE
Update README with postman instructions

### PR DESCRIPTION
**Pull request by Rumyana-Ch**
Some postman guidance to help people get set up faster. 

howitlooks

### CODE REVIEW

**Review by SamWilliamsGS — COMMENTED** [review]
Going to do a dry run following the guide so will review after that 👀

**Comment by Rumyana-Ch** [comment]
> Going to do a dry run following the guide so will review after that 👀

Yess thank you that will be great. But can you see the pictures?

File: README.md [hunk 1]
```diff
@@ -2,3 +2,35 @@
...
+
+4. Click on `import`.
+
+![image](https://user-images.githubusercontent.com/54809490/156166096-1576f482-21d2-41ba-b178-9c40aa4cdc02.png)
+
+5. Select `Folder` -> Upload the postman-configs folder. Click `Import`.
+
+![image](https://user-images.githubusercontent.com/54809490/156166533-da54a4ca-f2e2-4ad5-b848-6cf43df7e14f.png)
+
+6. Go to `Environments` and click on `api.gearset.com`. Configure using your `token` obtained from Gearset and the releveant job id. Can also change which environment we are targeting - local (localhost:2234-api.gearset.com), staging (staging-api.gearset.com) or app (api.gearset.com).
```
Comment (jwreford99) [inline README.md:22]: I don't think we want to mention Staging as this is a public repository so probably don't want to confuse folks. With that in mind also worth changing image to have the "normal" url?

**Review by jwreford99 — COMMENTED** [review]
Looks sensible, one thing is that this is for our PUBLIC git repo. If you wanted to just have this for local users you could add it to https://github.com/Gearset/public-api-postman??

Wider point is probably why do we have two versions in the first place

File: README.md (hunk 1 above)
Comment (SamWilliamsGS) [inline README.md:22]: I think `relevant` and also `localhost:2234-api.gearset.com` should be `localhost:2234`? 🙂

Comment (Rumyana-Ch) [reply README.md:22]: Haven't tried it yet, I will do in a sec

**Comment by Rumyana-Ch** [comment]
> Looks sensible, one thing is that this is for our PUBLIC git repo. If you wanted to just have this for local users you could add it to https://github.com/Gearset/public-api-postman??
> 
> Wider point is probably why do we have two versions in the first place

Oh wow sorry didn't realise this is public. Think I will just move it to the private repo then if you guys are okay with this? With that then we can leave the sensitive info as it is?

**Comment by SamWilliamsGS** [comment]
I'm wondering why this repo is public at all come to think of it. Do we want to encourage people messing around with our stuff on postman? Either way yeah this should go in the private repo. Good spot @jwreford99 I didn't see this!

File: README.md (hunk 1 above)
Comment (Rumyana-Ch) [reply README.md:22]: Yess you are right

**Comment by Rumyana-Ch** [comment]
Going to close this in favour of : https://github.com/Gearset/public-api-postman/pull/6

**Pull request closed without merging**